ajax/libs/imgareaselect/package.json

```diff
@@ -26,5 +26,13 @@
   "repository": {
     "type": "git",
     "url": "https://github.com/odyniec/imgareaselect"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/odyniec/imgareaselect.git",
+    "basePath": "distfiles",
+    "files": [
+      "**!(*.txt)/*"
```

Comment:
`"**/*.!(txt)"` will be better, you don't know when will the author add other files in `img/` or `js/`, I'll like to use rule with more flexible.